nucypher/config/utils.py

```diff
@@ -125,7 +125,7 @@ def check_config_tree(configuration_dir: str=None) -> bool:
 
 def check_config_runtime() -> bool:
     rules = (
-        (os.getuid() != 0, 'Cannot run as root user.'),
+        (os.name == 'nt' or os.getuid() != 0, 'Cannot run as root user.'),
```

Comment:
Presumably, ok if `Administrator`?

Reply:
For now it's not for real usage on Windows, so it's ok